google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -92,7 +92,7 @@ public OSProcessHandler createProcessHandler(
             }
 
             AppEngineStandardRunTask runTask =
-                new AppEngineStandardRunTask(runConfiguration);
+                new AppEngineStandardRunTask(runConfiguration, programRunner.getRunnerId());
```

Comment:
Can you explain how this runner id gets populated with "run" or "debug" like you mention in a comment in this PR? It's not clear to from reading the docs in ProgramRunner.